download/internalRewriteDownload/README.md

```diff
@@ -1,10 +1,19 @@
 Artifactory Internal Rewrite Download User Plugin
 =================================================
 
-*This plugin is currently being tested for Artifactory 5.x releases.*
-
 This plugin creates a virtual symlink called "latest", which redirects to the
 directory provided from the value of the `latest.folderName` property set on the
-root folder of the repository.
+root folder of the repository. 
+
+## Features
+In this example, this only works in the `dist-local` repository. However, this script will work for any local repo by:
+- Changing `.repositorySettings(new GenericRepositorySettingsImpl()).build()` to the package type you're using (e.g. if you are creating a Maven repository -> `.repositorySettings(new MavenRepositorySettingsImpl()).build()`
```

Comment:
@yamaguchim It seems this repository settings concerns the test case only, not the plugin itself. I believe this is not part of the plugin features.